v0/destinations/pipedrive/data/PipedriveProduct.json

```diff
@@ -0,0 +1,37 @@
+[
+  {
+    "destKey": "name",
+    "sourceKeys": "event",
+    "required": true
+  },
+  {
+    "destKey": "code",
+    "sourceKeys": "properties.code",
```

Comment:
please go through these
 https://docs.rudderstack.com/rudderstack-api-spec/rudderstack-ecommerce-events-specification/ordering#order-completed 
https://docs.rudderstack.com/rudderstack-api-spec/rudderstack-ecommerce-events-specification/ordering#product-viewed

and map.
for example code should be mapped with product_id/productId/sku

Reply:
unit --> quantity
tax --> tax
prices --> price

Reply:
okay will go through the ecom spec